alembic/versions/0c90faa0378f_create_algorithm_tables.py

```diff
@@ -40,13 +40,11 @@ def upgrade() -> None:
     sa.ForeignKeyConstraint(['algorithm_id'], ['algorithm.id'], ),
     sa.PrimaryKeyConstraint('id')
     )
-    op.create_index(op.f('ix_mpi_blocking_key_key'), 'mpi_blocking_key', ['key'], unique=False)
```

Comment:
These indexes were removed before the PR containing this migration was merged into main.  I'm deleting it here to resolve an error.